### PR TITLE
Remove Azure logging from jobs that log to ApplicationInsights

### PR DESCRIPTION
**Pull request by skofman1**
1. Removing Azure logging from jobs that already log to ApplicationInsight. The reasons for that is that we saw some instability in job run when there is a failure initializing Azure tracing ( https://github.com/NuGet/NuGetGallery/issues/3121). Also, this tracing is not user friendly (each line in different blob), so should be used only as if nothing else is available.

2. Fixed build of  Search.UpdateIndex job.

@maartenba @xavierdecoster @shishirx34 @scottbommarito

### CODE REVIEW

**Comment by shishirx34** [comment]
LGTM!

Just curious, HandlePackageEdits doesn't log to ApplicationInsights, does it?

**Comment by skofman1** [comment]
No, only new jobs support this. Eventually, we need to move all our stuff to log to AI

**Comment by scottbommarito** [comment]
LGTM!

File: src/Search.UpdateIndex/Search.UpdateIndex.csproj [hunk 1]
```diff
@@ -33,59 +33,59 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="EntityFramework, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, processorArchitecture=MSIL">
-      <HintPath>..\packages\EntityFramework.5.0.0\lib\net45\EntityFramework.dll</HintPath>
+      <HintPath>..\..\packages\EntityFramework.5.0.0\lib\net45\EntityFramework.dll</HintPath>
```
Comment (maartenba) [inline src/Search.UpdateIndex/Search.UpdateIndex.csproj]: Any reason this changed? (and other reference updates below as well)

**Comment by maartenba** [comment]
Minor (perhaps unimportant comment), otherwise :ship:

File: src/Search.UpdateIndex/Search.UpdateIndex.csproj (hunk 1 above)
Comment (skofman1) [inline src/Search.UpdateIndex/Search.UpdateIndex.csproj]: seems like the project was moved..

**Pull request merged**